### ISSUE TITLE
Probst bundles

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7186]
**Preferred term label:**
Probst bundles

**Synonyms**
Longitudinal callosal fascicles

**Definition (free text, please give PubMed ID)**
Probst bundles (also known as longitudinal callosal fascicles) are neuroanatomical homologues to the corpus callosum and can occur in association with callosal agenesis.  Probst bundles are white matter fibers that normally cross the corpus callosum but (because of the agenesis of the corpus callosum) fail to cross the midline, such that when they reach the corticoseptal boundary, they turn and run parallel to the interhemispheric fissure within the septal leaves, indenting the medial walls of the lateral ventricles 

Comment: The formation of Probst bundles is the most common structural anomaly associated with callosal dysgenesis. 

PMID:17898206
PMID:33642486

**Parent term (use hpo.jax.org/app)**
Abnormal corpus callosum morphology HP:0001273

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
612863 

PMID: 9863602  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
